log.go

```diff
@@ -73,3 +74,11 @@ func (logger *ZapEventLogger) Warningf(format string, args ...interface{}) {
 func FormatRFC3339(t time.Time) string {
 	return t.UTC().Format(time.RFC3339Nano)
 }
+
+func WithStamktrace(l *ZapEventLogger) *ZapEventLogger {
```

Comment:
`s/Stamk/Stack/` :)

stamks.jpg
![image](https://user-images.githubusercontent.com/3576549/127381283-c53ade54-d5e5-4d38-ba86-90090c0263d9.png)

Reply:
IDK what you are talking about ;p